tests/FileCard/index.js

```diff
@@ -0,0 +1,45 @@
+import { React, expect, sinon, shallow } from '../setup';
+import FileCard from '../../lib/FileCard';
+
+describe('FileCard', () => {
+  let wrapper;
```

Comment:
There is one, but it's only testing against the number of possible actions. I'll add more to it, but in any case these will have to be revisited once we add more functionality to them :)